changelog.d/10762.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where reactivated users would be missing from the user directory.
```

Comment:
We can probably just remove this one (but also be sure to remove it from the other branch).

Reply:
Not sure I follow you here (given that towncrier coalesces identical fragments). Is there a specific gain by tidying this up, or is it just nicer to have?

(Happy to clean up, but just anxious that it's further git-churn)

Reply:
It doesn't seem useful is all, it would end up pointing people to two identical PRs, which seems confusing.

I don't feel strongly either way, however.

Reply:
Oh, looking at the review page properly, I see what you mean: this PR adds two changelog files!!! (I thought you were referring to the two different files on two different PRs)

Reply:
+1 please remove the duplicate newsfile!